src/JsonSerializationVisitor.php

```diff
@@ -166,7 +166,7 @@ public function hasData(string $key): bool
     }
 
     /**
-     * @deprecated Use visitProperty(new StaticPropertyMetadata(null, 'name', 'value'), null) instead
+     * @deprecated Use `::visitProperty(new StaticPropertyMetadata(null, 'name', 'value'), 'value')` instead
```

Comment:
The first argument here is still null though which will cause the type checking in PHP 7 to fail again

Reply:
Good catch, fixed in https://github.com/schmittjoh/serializer/commit/80aec06a874ad196df64b371baa0d67689657596